common/sanctuary.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const Future        = require('fluture');
+const S             = require('sanctuary');
+const $             = require('sanctuary-def');
+
+
+//    ConcurrentFutureType :: Type -> Type -> Type
+const ConcurrentFutureType = $.BinaryType(
+  Future.Par['@@type'],
+  'https://github.com/fluture-js/Fluture#concurrentfuture',
+  x => S.type(x) === Future.Par['@@type'],
+  f => [],
+  f => []
+);
```

Comment:
Does this look right to you, @Avaq? It seems the ConcurrentFuture type lacks helpers corresponding to those that exist for defining `FutureType :: Type -> Type -> Type`. I'd love to be able to do this:

```javascript
const Future = require('fluture');
const $ = require('sanctuary-def');

const {ConcurrentFutureType, FutureType} = Future.defineTypes($);
```

Fluture could then provide type definitions *without* depending on sanctuary-def (although it would be a good idea to specify sanctuary-def as a peer dependency). What do you think?